tests/phpunit/includes/class-basic-object.php

```diff
@@ -13,22 +13,29 @@
  * @since 4.0.0
```

Comment:
Should an additional `@since` tag be added to note these changes?

Reply:
IMO, no, the `@since` is not needed as there is no functional change, no BC-break, nothing. The functionality hasn't changed in any perceivable  way when _using_ the `Basic_Object`, it is an internal refactor.